app/workers/pdf_report_worker.rb

```diff
@@ -40,5 +42,5 @@ def perform(service_id, account_id, period, reference)
 end
 
 if Rails.application.config.three_scale.daily_weekly_reports_pref
-  require_dependency 'pdf/report'
+  require 'pdf/report'
 end
```

Comment:
can we remove this require? Seems redundant given `Report` should be auto-loaded

Reply:
Removed